README.md

```diff
@@ -25,8 +25,8 @@ jobs:
     - uses: EddieHubCommunity/gh-action-open-source-labels@main
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
-        owner-name: 'EddieHubCommunity'
-        repository-name: 'gh-action-open-source-labels'
+       owner-name: ${{ github.repository_owner }}
+       repository-name: ${{ github.event.repository.name }}
```

Comment:
Could you add `one space` to make it look consistent and avoid `yaml` file errors!?

```suggestion
        owner-name: ${{ github.repository_owner }}
        repository-name: ${{ github.event.repository.name }}
```
#### Before changing
```yaml
    - uses: EddieHubCommunity/gh-action-open-source-labels@main
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
       owner-name: ${{ github.repository_owner }}
       repository-name: ${{ github.event.repository.name }}
```

#### After changing
```yaml
    - uses: EddieHubCommunity/gh-action-open-source-labels@main
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
        owner-name: ${{ github.repository_owner }}
        repository-name: ${{ github.event.repository.name }}
```

Reply:
Yes I will make the changes!